src/sort_cards.py

```diff
@@ -0,0 +1,35 @@
+"""Functions that sort lists of cards."""
+
+
+HEADS = {
+    'A': '0',
+    'T': '10',
+    'J': '11',
+    'Q': '12',
+    'K': '13'
+}
+
+
+def sort_cards(cards):
+    """Sort shuffled list of cards, sorted by rank.
+
+    >>> sort_cards(['3', '9', 'A', '5', 'T', '8', '2', '4', 'Q', '7', 'J', '6', 'K'])
+    ['A', '2', '3', '4', '5', '6', '7', '8', '9', 'T', 'J', 'Q', 'K']
+
```

Comment:
This works, but is pretty inefficient.  You are having to loop thru the deck several times to build these comprehensions, and then you use the nice sorted function with the key parameter later.  Why not just make the dict hold the rank for all things, or just the face/ace cards, and try using the number, except use face/ace rank?